### DISCUSSION TITLE
CTRL-Tab cycling between the tabs - cycles through tabs in recently used order

### DISCUSSION THREAD

**Discussion by ezhan** [PintaProject/Pinta#818] [Ideas]
Now there is no way to CTRL-Tab to next/previous tab.

And much needed: cycles through tabs in recently used order (to compare two tabs), as it is in any DE & Firefox.

**Comment by cameronwhite** [comment]
See #594 - this will be fixed for the next release

It will just be in next/previous order though. Offhand I'm not sure how easy it would be to add recently used order on Pinta's end, since we use the tab bar from libadwaita (https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.TabBar.html) which handles the Ctrl+Tab shortcut itself
